DependencyInjection/Configuration.php

```diff
@@ -157,6 +157,7 @@
  *   - [level]: level name or int value, defaults to DEBUG
  *   - [bubble]: bool, defaults to true
  *   - [ident]: string, defaults to
+ *   - [rfc]: int 0 (RFC3164) or 1 (RFC5424), defaults to 1
```

Comment:
please list the constants from \Monolog\Handler\SyslogUdpHandler instead of plain int

Reply:
Got it. In any case, the user should understand that monolog.yaml should look like

```
monolog:
    handlers:
        syslog:
            type: syslogudp
            host: 127.0.0.1
            port: 514
            level: error
            ident: app
            rfc: 0
```